src/sentry/api/endpoints/sentry_apps_stats.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases import SentryAppsBaseEndpoint
+from sentry.models import SentryApp
+from sentry.api.permissions import SuperuserPermission
+
+
+class SentryAppsStatsEndpoint(SentryAppsBaseEndpoint):
+    permission_classes = (SuperuserPermission, )
+
+    def get(self, request):
+        sentry_apps = SentryApp.objects.all().prefetch_related('installations')
```

Comment:
Will this load all the installation records? If you only need the counts you could use the [annotate method](https://docs.djangoproject.com/en/1.7/topics/db/aggregation/#generating-aggregates-for-each-item-in-a-queryset) to get the counts without hydrating records.